CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# How to Contribute
+
+Here are some important resources:
+
+  * [Leanplum Docs](http://leanplum.com/docs) The official Leanplum docs.
+  * Support: Get help at support@leanplum.com.
+
+## Submitting changes
+
+Please consider following guidelines before creating a PR:
+
+- Confirm the intention and design of your change by reaching out to Leanplum Dev Team through a new Github Issue.
```

Comment:
Commented on Android on but shall we change this to:
reaching out to your CSM or to Leanplum Dev Team through a new Github Issue.